File: Medical_Profile/department.cs

namespace Medical_Profile
{
 public class Departments_Return
 {
  public string token { get; set; }
  public string next { get; set; }
  public int totalcount { get; set; }
  public Dept_Return[] departments { get; set; }
 }

 public class Dept_Return
 {
  public string state { get; set; }
  public string departmentid { get; set; }
  public string address { get; set; }
  public string name { get; set; }
  public string zip { get; set; }
  public string city { get; set; }
  public string fax { get; set; }
  public string providergroupname { get; set; }
  public string phone { get; set; }
 }
}

File: Medical_Profile/department.vb
Imports System

Public Class Departments_Return
    Public Property token As String
    Public Property [next] As String
    Public Property totalcount As Integer
    Public Property departments As Dept_Return()
End Class

Public Class Dept_Return
    Public Property state As String
    Public Property departmentid As String
    Public Property address As String
    Public Property name As String
    Public Property zip As String
    Public Property city As String
    Public Property fax As String
    Public Property providergroupname As String
    Public Property phone As String
End Class
